tests/framework/installer/ceph_installer.go

```diff
@@ -65,7 +65,7 @@ mon_data_avail_warn = 10
 [mon]
 mon compact on start = true
 `
-	volumeReplicationVersion = "v0.5.0"
+	volumeReplicationVersion = "v0.12.0"
```

Comment:
The CI is passing with this update, so I assume no other changes needed to the tests.

Reply:
Ah, was this since #10777 merged? 
@yati1998 Would you like to remove this related volume replication test code since you added it originally? thanks